code/modules/clothing/suits/jobs.dm

```diff
@@ -151,6 +151,13 @@
 	inhand_icon_state = "officerbluejacket"
 	body_parts_covered = CHEST|ARMS
 
+/obj/item/clothing/suit/toggle/security/casualofficer
```

Comment:
there doesnt seem to be a subtype like toggle/security

Reply:
there isn't but there should be